lib/Storage/MySqlTimeSeriesStore.php

```diff
@@ -126,7 +136,7 @@ public function getStatsReport($fromDt, $toDt, $displayIds, $layoutIds, $mediaId
 
         $body .= ' WHERE stat.type <> \'displaydown\'
                 AND stat.end > :fromDt
-                AND stat.start <= :toDt
+                AND stat.start <= DATE_ADD(:toDt, INTERVAL 1 DAY)
```

Comment:
I'm not clear why we are adding a day to the toDt here?

If we do need to do that, then should we do that in PHP rather than MySQL?

Reply:
We don't see any stat records of to date in the grid if we don't add a date. I will change it to PHP date.